### PR TITLE
Pytorch 10.2 seq2seq attention

### PR DESCRIPTION
**Pull request by ypandya**
*Description of changes:*

Pytorch port for 10.2. Sequence to Sequence with Attention Mechanisms.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_attention-mechanisms/seq2seq-attention.md [hunk 1]
```diff
@@ -86,7 +129,21 @@ encoder.initialize()
...
+
+```{.python .input}
+#@tab pytorch
+encoder = d2l.Seq2SeqEncoder(vocab_size=10, embed_size=8,
+                             num_hiddens=16, num_layers=2)
+encoder.eval()
+decoder = Seq2SeqAttentionDecoder(vocab_size=10, embed_size=8,
+                                  num_hiddens=16, num_layers=2)
+decoder.eval()
+X = torch.zeros((4, 7), dtype=torch.long)
```
Comment (AnirudhDagar) [inline chapter_attention-mechanisms/seq2seq-attention.md:146]: Use d2l.zeros

**Comment by mli** [comment]
Job d2l-en/PR-1122/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1122/

File: chapter_attention-mechanisms/seq2seq-attention.md (hunk 1 above)
Comment (ypandya) [reply chapter_attention-mechanisms/seq2seq-attention.md]: Updated

**Comment by mli** [comment]
Job d2l-en/PR-1122/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1122/

**Pull request merged**